### PR TITLE
Add model

### LINKED ISSUE #12: Add model to the app.

### PR DESCRIPTION
**Pull request by thibautjombart**
Changes include:

* added a bunch of `.R` files implementing: prediction of admissions, prediction of bed occupancy, plots of results, auxiliary scripts
* moved all R files to `R/`
* added model to the app
* removed placeholder graphs from the app
* removed placeholder R files

This is a minimal functional product. Will add a couple of issues for improvements for the first release.

### CODE REVIEW

**Comment by erees** [comment]
Screenshot 2020-03-26 at 09 03 22

Just testing out the app on my computer. Seems to be running fine. One thing - when you put a lower doubling time (1.5) and keep the uncertainty in doubling times at default (1) the credible intervals become so wide the graph is no longer usefull (added screen shot to show what I mean). I think Sam may have mentioned a solution to it? It's just from reading a doubling time of ~2 has been reported for many countries so it's likely many people will try these and may be worth doing something before first release. Sorry if this is already being worked on and I've missed it

**Comment by mert0248** [comment]
I get this error:

Screenshot 2020-03-26 at 09 42 11

**Comment by thibautjombart** [comment]
> Screenshot 2020-03-26 at 09 03 22
> 
> Just testing out the app on my computer. Seems to be running fine. One thing - when you put a lower doubling time (1.5) and keep the uncertainty in doubling times at default (1) the credible intervals become so wide the graph is no longer usefull (added screen shot to show what I mean). I think Sam may have mentioned a solution to it? It's just from reading a doubling time of ~2 has been reported for many countries so it's likely many people will try these and may be worth doing something before first release. Sorry if this is already being worked on and I've missed it

Indeed, we started discussing it on slack. This may not be for the first release, so I am creating a dedicated issue:
https://github.com/thibautjombart/covid19_bed_occupancy/issues/16

Quick fix might be just using a small error by default.

**Comment by thibautjombart** [comment]
> I get this error:
> 
> Screenshot 2020-03-26 at 09 42 11

You might be using an outdated version of `tidyr`. I am using `tidyr_1.0.2` for R 3.6.3. Can you check your R version, maybe run `update.packages(ask = FALSE)` and try again?

**Comment by erees** [comment]
> I get this error:
> 
> Screenshot 2020-03-26 at 09 42 11

@mert0248 I also got this error at first. I update tidyr package and then it worked. I think pivot_longer is a newer release

**Comment by thibautjombart** [comment]
Cool @erees @mert0248 if you are happy with replies can you accept the PR so I can merge it?

File: app/R/plot_beds.R [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#' r_duration <- function(n = 1) rexp(n, .2)
+#'
+#' ## get daily bed needs predictions
+#' beds <- predict_beds(x$date, x$mean, r_duration)
+#' beds
+#' plot_beds(beds)
+
+plot_beds <- function(x, ...) {
+  plot(x,
+       quantiles = c(.025, .5),
```
Comment (esnightingale) [inline app/R/plot_beds.R:31]: Do you mean to have 0.025 and 0.975 quantiles here?

**Comment by erees** [comment]
> Cool @erees @mert0248 if you are happy with replies can you accept the PR so I can merge it?

Yes I'm happy - I jsut can't work out how to accept it

**Review by erees — APPROVED** [review]

File: app/R/plot_beds.R (hunk 1 above)
Comment (mert0248) [reply app/R/plot_beds.R:31]: @thibautjombart I'm having trouble with R and package updates - happy to approve reply if this is holding things up - where/how do I do that? Thanks

Comment (samclifford) [reply app/R/plot_beds.R:31]: @esnightingale I had a similar question but digging around it seems that the plot method generates a full list of quantiles to be requested by taking `quantiles` and forming `1-quantiles`.

**Pull request merged**